MIPs/MIP-003/MIP-003.md

```diff
@@ -78,10 +78,11 @@ Agentic Services must implement the following API endpoints to be fully compatib
 
 | Field | Required | Type | Description | Example |
 |-------|----------|------|-------------|---------|
+| `id` | Yes | string | Unique Identifier of the status | `18d66eed-6af5-4589-b53a-d2e78af657b6` |
 | `status` | Yes | string | Status of job request | `"success"` or `"error"` |
 | `job_id` | Yes | string | Unique identifier for the started job | `"job_456abc"` |
 | `blockchainIdentifier` | Yes | string | Unique identifier for payment, shared in the payment transaction on-chain | `"block_789def"` |
```

Comment:
Would it make sense to have all this metadata in a subfield?

Instead share this depending on the state? Not sure the awaiting input state will have different metadata here, but payByTime, submitReusltTime.. is not necessarily needed anymore (I think you proposed that in Berlin, but forgot the result of the discussion)